### DISCUSSION TITLE
Synology Docker config.yml Not Found

### DISCUSSION THREAD

**Discussion by beekness** [meisnate12/Plex-Meta-Manager#690] [Q&A]
I have searched and tried everything I could for hours to get the Config.Xml to be found by PMM in Docker on my Synology NAS.

Where am I supposed to save the Config file?

I created a folder inside of my docker folder and saved it there and added the folder to the volume during installation. But I am not sure how to point PMM to look there for it.  Below is the error I get in the log when I start PMM (which crashed shortly thereafter, which I assume is because it needs the config file)

Traceback (most recent call last):
File "//plex_meta_manager.py", line 72, in <module>
elif not os.path.exists(os.path.join(default_dir, "config.yml")):   raise util.Failed(f"Config Error: config not found at {os.path.abspath(default_dir)}")
modules.util.Failed: Config Error: config not found at //config

Has anyone created a step by step install for Synology Nas, all I could find online was for unraid.

thank you for whomever can help me out on this one because I am stumped..

**Comment by meisnate12** [comment]
you need to map a folder to the /config directory also its a .yml file not a .xml

**Reply by beekness** [reply]
hello @meisnate12, thank you for jumping in.,

yes I have it as yml not xml, was a long day yesterday. I have the yml files in a folder called MetaManager in the docker folder, I have mapped the confirg.yml file and MetaManager folder in docker in the volume tab also but the error persists. It still cannot find the config.yml

**Reply by meisnate12** [reply]
You don't map the config file just the config folder

**Reply by beekness** [reply]
Ok I removed the yml file mapping and only have the MetaManager Folder with the 3 yml files in there mapped but I am still getting the can not find config.yml file. Is there another setting that needs to be done in docker to point the program to the config? Possibly in the environment tab?

I just started using docker a few months ago and I am still in the beginner phase of understanding it.

**Reply by beekness** [reply]
any thoughts on how to get this working on a Synology NAS docker setup?

**Reply by meisnate12** [reply]
take a screen shot of what your mappings look like and ill see what i can figure out

**Reply by beekness** [reply]
thank you for your help, i figured out that when mapping the config i needed to put /config/, /config was not enough.

**Reply by AirRock187** [reply]
@beekness I'm getting the same error, can I see what your Volume parameters look like?
